README.md

```diff
@@ -1,6 +1,6 @@
 - Custom Locale Plugin
-- Version: 1.0.1
-- Author: Carola Fanselow (updates by Alec Smecher)
+- Version: 1.0.2
```

Comment:
Let's make this version 1.1.0 since it's a significant update. Same for the `version.xml` file.

Reply:
Sounds good to me. Updated this in `pluginDocumentation_customLocale.md` as well.